plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -5137,6 +5138,17 @@ Actual rows (up to 100 of 1 extra rows shown, 1 rows in total):
                             [1, 2]""");
     }
 
+    @Test
+    public void testDropColumnWithUnsupportedSerDe()
+    {
+        for(HiveStorageFormat format : ImmutableSet.of(HiveStorageFormat.AVRO, HiveStorageFormat.JSON, HiveStorageFormat.RCBINARY)) {
```

Comment:
Instead of wrapping inside a for-loop - Can we have a static method which asserts if the query fails and invoke them here individually ?